### ISSUE TITLE
Removal of right click folder - random bookmark

### ISSUE THREAD

**Issue by SHHSSH** [PikadudeNo1/RandomBookmark#1]
There is not efficient access to a random bookmark. Not even a hotkey to prompt open RandomBookmark from the toolbar. I've been an avid user for a good year or more now, please revert back to the capability of right-clicking on a specific folder.

**Comment by polyzen** [comment]
It's most likely not currently possible with WebExtensions. Glad to see this updated!

**Comment by SHHSSH** [comment]
Similarly, very thankful for an update. But you're right @polyzen, it may be a Quantum issue.

**Comment by PixievoltNo1** [comment]
Indeed, implementing the old interface has not been possible since Firefox Quantum. I would like to help you use RBFF efficiently, and I can add a hotkey. I'd like to hear any other ideas you have.

Closing this issue, but further comments are welcome.

**Comment by PixievoltNo1** [comment]
Good news: I have recently learned that Firefox now allows WebExtensions to add context menu items for bookmark folders! But there's a catch: context menu items must be added for folders *and* individual bookmarks, not just one or the other, which means RBFF's items would appear for right-clicks where it makes no sense.

So here's the plan: in version 2.1, the context menu items will be back, but off by default. If turned on but then used on an individual bookmark, Random Bookmark From Folder will ask if you'd like to use it on the parent folder.

**Comment by PixievoltNo1** [comment]
This was ultimately not added for version 2.1, but version 3.0 *will* have this, appearing on folders only thanks to improvements in Firefox, and on by default. Thanks for your patience.
